content/api/events/events_post.md

```diff
@@ -11,11 +11,11 @@ This end point allows you to post events to the stream. Tag them, set priority a
 ##### ARGUMENTS
 * `title` [*required*]:  
     The event title. *Limited to 100 characters.*  
-    Use `msg_title` with ruby librairy.
+    Use `msg_title` with [the Datadog ruby library](https://github.com/DataDog/dogapi-rb).
```

Comment:
Ruby should be capitalized.

+    Use `msg_title` with the Datadog [Ruby library](https://github.com/DataDog/dogapi-rb).